src/elements/play-pen/play-pen.ts

```diff
@@ -409,4 +447,13 @@ export class PlayPen extends LitElement {
       this._uploaded =
         this.#bundleStore?.upload(this._bundle) ?? Promise.resolve({})
   }
+
+  #assetsUpdated = () => {
+    if (this._assetFilesystem !== this._assets.filesystemType) {
+      this._assetFilesystem = this._assets.filesystemType
+    }
+    if (this._src) {
```

Comment:
I didn't understand why we're gating the call to #setSrc() and couldn't find other patterns following that.

Reply:
`_src` is `string | undefined` and `#setSrc` does not accept undefined as its first argument.  An alternative could be:

```ts
this.#setSrc(this._src ?? '', false)
```